src/main/java/seedu/address/model/person/CheckIfBirthday.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.person;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code ReadOnlyPerson}'s {@code Birthday month and day} matches today.
+ */
+public class CheckIfBirthday implements Predicate<ReadOnlyPerson> {
+
+    public CheckIfBirthday(){ }
+    /**
+     * Checks if today's day and month matches a person's birthday
+     * @param person
+     * @return
+     * @throws ParseException
```

Comment:
Rather than use an empty `@return/@param/@throws`, either give a description as well, or delete it altogether.

Reply:
Hi Prof,
By description do you mean description of what the function does? Or description about the parameter/what the method throws etc

Reply:
The latter. Lines 17-19 don't add any value to the comment as those information are already in the method signature.

Reply:
Oh yes I see. Will delete them now. Thanks prof!